src/modules/Dashboard/state/Applets/Applets.reducer.ts

```diff
@@ -12,15 +13,15 @@ import {
 export const reducers = { deleteApplet };
 
 export const extraReducers = (builder: ActionReducerMapBuilder<AppletsSchema>): void => {
-  createAppletsPendingData({ builder, thunk: getApplets, key: 'applets' });
   createAppletsPendingData({ builder, thunk: getApplet, key: 'applet' });
+  createAppletsPendingData({ builder, thunk: getWorkspaceApplets, key: 'applets' });
   createAppletsPendingData({ builder, thunk: getEvents, key: 'events' });
 
-  createAppletsFulfilledData({ builder, thunk: getApplets, key: 'applets' });
+  createAppletsFulfilledData({ builder, thunk: getWorkspaceApplets, key: 'applets' });
   createAppletsFulfilledData({ builder, thunk: getApplet, key: 'applet' });
   createAppletsFulfilledData({ builder, thunk: getEvents, key: 'events' });
 
-  createAppletsRejectedData({ builder, thunk: getApplets, key: 'applets' });
   createAppletsRejectedData({ builder, thunk: getApplet, key: 'applet' });
+  createAppletsRejectedData({ builder, thunk: getWorkspaceApplets, key: 'applets' });
```

Comment:
Is it OK using 'applets' key for workspaceApplets? As I can see there is getAppletsApi

Reply:
yes, I set applets